src/Aspire.Dashboard/Model/ResourceEndpointHelpers.cs

```diff
@@ -11,7 +11,7 @@ internal static class ResourceEndpointHelpers
     /// <summary>
     /// A resource has services and endpoints. These can overlap. This method attempts to return a single list without duplicates.
     /// </summary>
-    public static List<DisplayedEndpoint> GetEndpoints(ResourceViewModel resource, bool includeInternalUrls = false)
+    public static List<DisplayedEndpoint> GetEndpoints(ResourceViewModel resource, bool includeInternalUrls = false, bool includeNonEndpointUrls = false)
```

Comment:
Rename type to ResourceUrlHelpers. And method to GetUrls. And DisplayedEndpoint to DisplayedUrl. etc